src/Shell/Task/SimpleBakeTask.php

```diff
@@ -90,7 +90,7 @@ public function main($name = null)
      * Generate a class stub
      *
      * @param string $name The classname to generate.
-     * @return void
+     * @return string
```

Comment:
this method has a `return;` => that should actually be `return null;` then and the doc block should also say `|null`.